integration-tests/src/test/java/org/zowe/apiml/functional/apicatalog/ApiCatalogAuthenticationTest.java

```diff
@@ -148,6 +158,7 @@ void givenValidBasicAuthenticationAndCertificate(String endpoint, Request reques
                         endpoint
                     )
                     .then()
+                    .log().all()
```

Comment:
should we keep log all enabled?

Reply:
this is useful for troubleshooting in general, I'd keep it